### DISCUSSION TITLE
Lab: Coding Tools and Best Practices

### DISCUSSION THREAD

**Discussion by mbcann01** [brad-cannell/intro.r.public#28] [Labs]
Please feel free to post questions about this lab below. You are also welcome to leave comments below about the usefulness of this lab. Which parts were helpful? What could I do to improve it? What is still unclear?

**Comment by OJONESD** [comment]
Render Error:
At the end of the lab I got an error after clicking Render. It says I am missing packages "knitr" and "rmarkdown", do I need to install these packages to be able to convert the .qmd file to html? 
Thank you for taking the time to create this lab it was very detailed and the example Quarto file was helpful to reference to see where I messed up.
Olivia Jones

**Reply by mbcann01** [reply]
Hi @OJONESD 

Yes, you are correct. You just need to install `knitr` and `rmarkdown` on your computer. You can do that by using the `install.packages()` command in the RStudio console pane. I will also add this to the lab instructions.
